README.md

```diff
@@ -49,44 +49,56 @@ The documentation contains information about how to setup and run the two planne
 
 ### Installation
 
-This is a step-by-step guide to install and build all the prerequisites for running this module on Ubuntu 16.04 and Kinetic - Ubuntu 18.04 and Melodic are supported as well. You might want to skip some of them if your system is already partially installed.
-
-Note that in the following instructions, we assume your catkin workspace (in which we will build the avoidance module) is in `~/catkin_ws`, and the PX4 Firmware directory is `~/Firmware`. Feel free to adapt this to your situation.
-
-1. Add ROS to sources.list.
-
-   ```bash
-   echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list
-   sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
-   sudo apt update
-   ```
-
-1. Install gazebo with ROS (use ROS melodic based on preference).
-
-   ```bash
-   sudo apt install ros-kinetic-desktop-full
-
-   # Source ROS
-   source /opt/ros/kinetic/setup.bash
-   ```
-
-  Full installation of ROS Kinetic comes with Gazebo 7.
-
-  If you are using different version of Gazebo,
-
-  please make sure install ros-gazebo related packages
-
-  For Gazebo 8,
-  ```
-  sudo apt install ros-kinetic-gazebo8-*
-  ```
-  For Gazebo 9,
-  ```
-  sudo apt install ros-kinetic-gazebo9-*
-  ```
+This is a step-by-step guide to install and build all the prerequisites for running the avoidance module on either:
+- **Ubuntu 18.04:** *ROS Melodic* with Gazebo 9 (preferred).
+- **Ubuntu 16.04:** *ROS Kinetic* with Gazebo 7
+You might want to skip some steps if your system is already partially installed.
+
+> **Note:** These instructions assume your catkin workspace (in which we will build the avoidance module) is in `~/catkin_ws`, and the PX4 Firmware directory is `~/Firmware`.
+  Feel free to adapt this to your situation.
+
+1. Add ROS to sources.list:
+   * Melodic
+     ```bash
+     sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+     sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+     sudo apt update
+     ```
+   * Kinetic
+     ```bash
+     echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list
+     sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+     sudo apt update
+     ```
+
+1. Install ROS with Gazebo:
+   * ROS Melodic (with Gazebo 9)
+     ```bash
+     sudo apt install ros-melodic-desktop-full
+
+     # Source ROS
```

Comment:
Note how this is different from the kinetic version. Essentially this sets the ROS environment for any prompt you launch. I think that otherwise you need to set it every time you start a prompt?

Apparently you can do this unless you need to support multiple versions of ROS.
Thoughts?

Reply:
yes correct. I think it's fine to suggest to ad it to the bashrc.